esbuildConfig.js

```diff
@@ -0,0 +1,122 @@
+const { sassPlugin, postcssModules } = require('esbuild-sass-plugin');
+const autoprefixer = require('autoprefixer');
+const postcss = require('postcss');
+const yargsParser = require('yargs-parser');
+const { writeFileSync } = require('fs');
+const { resolve, join } = require('path');
+
+// from https://github.com/bvaughn/react-virtualized/issues/1212#issuecomment-847759202 workaround for https://github.com/bvaughn/react-virtualized/issues/1632 until it is released.
+const resolveFixup = {
+    name: 'resolve-fixup',
+    setup(build) {
+        build.onResolve({ filter: /react-virtualized/ }, async (args) => {
```

Comment:
Please change this to 
```
setup(build) {
        build.onResolve({ filter: /react-virtualized/ }, async (args) => {
            return {
                path: require.resolve('react-virtualized/dist/umd/react-virtualized.js')
            };
        });
    }
```
So that it still works even when `react-virtualized` is in a different `node_modules` folder.

Reply:
added last time i tried this but it resolved to commonjs, now it worked locally for me
<img width="685" alt="image" src="https://github.com/SAP/open-ux-tools/assets/103586139/e523cdd9-85a8-40e0-802e-41fe45d3f6b4">